packages/react-component-library/src/components/Avatar/partials/StyledAvatar.tsx

```diff
@@ -0,0 +1,35 @@
+import styled, { css } from 'styled-components'
+import { selectors } from '@royalnavy/design-tokens'
+
+interface StyledAvatarProps {
+  $dark?: boolean
+}
+
+const { color } = selectors
+
+export const StyledAvatar = styled.span<StyledAvatarProps>`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 30px;
+  width: 30px;
+  border-radius: 15px;
+  background: ${color('neutral', 'white')};
+  color: ${color('neutral', '600')};
+  font-size: 12px;
+  font-weight: bold;
+  text-decoration: none;
+
+  &:hover {
+    background: ${color('action', '500')};
+    color: ${color('neutral', 'white')};
+    cursor: pointer;
+  }
+
+  ${({ $dark }) =>
```

Comment:
I am wondering whether this pattern is consistent with other variants. ie should we have `$variant` and pass through `dark` which then uses a lookup?

Appreciate it is just one at this stage.

Reply:
>Appreciate it is just one at this stage.

I felt like this was the simplest implementation at this stage.